internal/keymagic/publicKeyStore.go

```diff
@@ -3,26 +3,45 @@ package keymagic
 import (
 	"context"
 	"fmt"
-	"github.com/quan-to/chevron/internal/config"
-	"github.com/quan-to/chevron/internal/database"
 	"github.com/quan-to/chevron/internal/tools"
 	"github.com/quan-to/chevron/pkg/models"
 
 	"github.com/quan-to/slog"
```

Comment:
Same comment for [imports](https://github.com/golang/go/wiki/CodeReviewComments#imports).